.circleci/config.yml

```diff
@@ -254,6 +262,9 @@ jobs:
           disable_synthetic: true
           enable_native_custom_element_lifecycle: true
           enable_scoped_custom_element_registry: true
+      - run_karma:
+          disable_synthetic: true
+          disable_aria_reflection_polyfill: true
```

Comment:
I don't see the point in testing all possible combinations here. I'm using `disable_synthetic: true` to make the test more "pure," but the tests should pass just as easily with it set to `false`. ARIA reflection has to relation to synthetic shadow.